app/models/viewmodels/notificationspage/converters/ControlResponseConverter.scala

```diff
@@ -57,16 +58,22 @@ class ControlResponseConverter @Inject()(decoder: Decoder, dateTimeFormatter: Da
   private def buildActionCodeExplanation(actionCode: String)(implicit messages: Messages): Option[String] =
     decoder.actionCode(actionCode).map(code => paragraph(messages(code.contentKey)))
 
-  private def buildErrorsExplanation(errorCodes: Seq[String])(implicit messages: Messages): String = {
+  private def buildILEErrorsExplanation(errorCodes: Seq[String])(implicit messages: Messages): String = {
     val errorsExplanation = errorCodes.flatMap { code =>
-      val errorCode = decoder.errorCode(code)
+      val errorCode = decoder.ileErrorCode(code)
       if (errorCode.isEmpty) logger.info(s"Received inventoryLinkingControlResponse with unknown error code: $code")
```

Comment:
Please, create some kind of pattern match on errors, because currently we will be logging all CHIEF codes.

Reply:
Fixed, good spot